src_erl/NerlnetApp/src/MainServer/actionHandler.erl

```diff
@@ -19,6 +19,7 @@ init(Req0, [Action, Main_genserver_Pid]) ->
   {_,Body,_} = cowboy_req:read_body(Req0),
   % io:format("mainserver got ACTION: ~p~n",[Body]),
   case Action of
+    clientsPhaseUpdate -> gen_statem:cast(Main_genserver_Pid, {clientsPhaseUpdate,Body});
     clientsTraining ->  gen_statem:cast(Main_genserver_Pid, {clientsTraining,Body});
```

Comment:
@GuyPerets106 
These clientsTraining and clientsPredict should be deprecated once phase update works.
Is it correct?

Reply:
Yes , currently the query which change between these phases is clientsPhaseUpdate